baram/efs_manager.py

```diff
@@ -0,0 +1,88 @@
+import boto3
+
+from baram.log_manager import LogManager
+
+
+class EFSManager(object):
+    def __init__(self):
+        self.cli = boto3.client('efs')
+        self.logger = LogManager.get_logger()
+
+    def list_file_systems(self):
+        """
+        List one or more of your EFS.
+
+        :return:
+        """
+        return self.cli.describe_file_systems()['FileSystems']
+
+    def list_mount_targets(self, file_system_id: str):
+        """
+        List mount targets of specific file system.
+
+        :param file_system_id: FileSystemId
+        :return:
+        """
+        try:
+            return self.cli.describe_mount_targets(FileSystemId=file_system_id)['MountTargets']
+        except:
+            self.logger.info('error')
+            return []
+
+    def delete_file_systems(self, redundant_fs_ids: list = []):
+        """
+        Delete file systems of disused domains.
+
+        :param redundant_fs_ids: list of FileSystemId.
+        :return:
+        """
+        for fs_id in redundant_fs_ids:
+            mount_target_ids = [mt['MountTargetId'] for mt in self.list_mount_targets(fs_id)]
+
+            for mt_id in mount_target_ids:
+                self.delete_mount_targets(mt_id)
+
+            while True:
```

Comment:
Corner case is too dangerous. You should avoid unstoppable program running.

Reply:
I'll fix this dangerous case.